README.md

```diff
@@ -18,12 +20,73 @@ or
 npm install @react-native-community/art --save
 ```
 
-### Automatic linking
+## Linking module
 
-```sh
+### Mostly automatic linking
```

Comment:
maybe mention autolinking as a preferred way?

Reply:
Is there any documentation that I can link to? `0.60.0` is not released yet, so we can add it later on.

Reply:
Just running `pod install`, but yea, it's not stable yet, so let's postpone